spark/processing/3.0/py3/docker/py37/Dockerfile.cpu

```diff
@@ -108,6 +108,6 @@ WORKDIR $SPARK_HOME
 
 # Install the sagemaker feature store spark connector
 # https://docs.aws.amazon.com/sagemaker/latest/dg/batch-ingestion-spark-connector-setup.html
-RUN /usr/bin/python3 -m pip install sagemaker-feature-store-pyspark-3.0 --no-binary :all:
+RUN /usr/bin/python3 -m pip install sagemaker-feature-store-pyspark-3.0==1.1.2 --no-binary :all:
```

Comment:
Can you link the PySpark version for feature store?  is 1.12 compatible for all the Spark Images from 3.0 all the way to 3.3?

Reply:
Yes, we support the Spark versions from 3.0 ~ 3.3

https://pypi.org/project/sagemaker-feature-store-pyspark-3.0/
https://pypi.org/project/sagemaker-feature-store-pyspark-3.1/
https://pypi.org/project/sagemaker-feature-store-pyspark-3.2/
https://pypi.org/project/sagemaker-feature-store-pyspark-3.3/

Reply:
And maven repo is here:

https://mvnrepository.com/artifact/software.amazon.sagemaker.featurestore/sagemaker-feature-store-spark-sdk_2.12/1.1.2